06_puppy/sahidakhtar1/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+type Puppy struct {
+	ID                   uint
+	Breed, Colour, Value string
+}
+
+type Storer interface {
+	CreatePuppy(Puppy)
+	ReadPuppy(uint) Puppy
+	UpdatePuppy(uint, Puppy)
+	DeletePuppy(uint) bool
+}
+
+//Map Store Starts
```

Comment:
I'd create 3 more files for this:
`storer.go`,  `mapstore.go` and `sunystore.go`.
As long as they are all in the same directory and contained in package `main` it shouldn't be an issue.

Reply:
Created separate files for each storer and its corresponding  testing file.